packages/js/components/src/form/form.tsx

```diff
@@ -262,7 +262,7 @@ function FormComponent< Values extends Record< string, any > >(
 			}
 
 			if ( callback ) {
-				callback( values );
+				return callback( values );
```

Comment:
Why this change? 
Splitting the changes into more than 1 commit and making the commits bit more more descriptive, would make the review easier.

Reply:
> Why this change?

When submitting the form we need a feedback in order to complete other things like dealing with server error or success mesages. With this we can do something like `onSubmit().then(...).catch(...)` to handle post submit tasks.

 > Splitting the changes into more than 1 commit and making the commits bit more more descriptive, would make the review easier.

You are right! I'll do this always from now. Thanks!!!

Reply:
Thanks for the answer!